src/ui/RadioGroup/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { withDesign } from 'storybook-addon-designs';
+
+import { Flex, FormLabel } from 'ui';
+
+import {
+  RadioGroupIndicator,
+  RadioGroupRadio,
+  RadioGroupRoot as RadioGroupRootComponent,
+} from './RadioGroup';
+
+export default {
+  title: 'Design System/Components/RadioGroup',
+  component: RadioGroupRootComponent,
+  decorators: [withDesign, Story => <Story />],
+  argTypes: {
```

Comment:
this is nice... to remove all the noisy options around props that are not specific... is there a way to use shared argTypes across many components? Or maybe have a helper function for hiding these?

Reply:
https://storybook.js.org/docs/react/api/argtypes#global-argtypes

Reply:
Tried to move this to `preview` but didn't seem to work...

Reply:
It work for me, perhaps you need to restart storybook, done with https://github.com/coordinape/coordinape/pull/1594/commits/ee28e494e13e1eec958aeb4acec2b91d091f2710